Automattic/vip-go-mu-plugins
# Add Jetpack 10.3 (beta)

## Description

Adding new JP version: https://github.com/Automattic/jetpack-production/releases/tag/10.3-beta

## Changelog Description

### Plugin Added: Jetpack 10.3

We have added Jetpack 10.3-beta. If you want to try this beta, you need to pin your site to use Jetpack 10.3 by adding this line to your vip-config.php:

```
define( 'VIP_JETPACK_PINNED_VERSION', '10.3' );
```

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Pin your testing site to the new subtree version: `define( 'VIP_JETPACK_PINNED_VERSION', '10.3' );`
1. Go to the Jetpack dashboard in wp-admin and ensure version 10.3-beta is active
1. Ensure everything else works as expected
